Reneiski_Homework1/ReneiskiSelenium11/Tests/Tests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using NUnit.Framework;
+using ReneiskiSelenium11.Common.Extensions;
+using ReneiskiSelenium11.Data;
+using ReneiskiSelenium11.Data.Constants;
+using ReneiskiSelenium11.PageObjects;
+using ReneiskiSelenium11.PageObjects.Pages;
+
+namespace ReneiskiSelenium11.Tests
+{
+    public class DemoQATests : BaseTest
+    {
+
+        [Test]
+        public void FirstCheckBoxTest()
+        {
+            var check = new CheckBoxPage();
+
```

Comment:
do you really need empty lines after every line? usually, an empty line is needed in test to split logically different blocks of code

Reply:
Fixed